app/config-schema.js

```diff
@@ -322,5 +322,23 @@ module.exports = {
         default: true
       }
     }
+  },
+  whitelabel: {
+    availableInFrontend: true,
+    bgImage: {
+      doc: 'The background image URL',
+      format: String,
+      default: ''
```

Comment:
Perhaps this should default to the URL of the default image we currently use. Then we'd always read this value from `config.get` and we'd either get the path to whatever image should be used.

Reply:
Sounds good to me